features/step_definitions/us152_search_for_deed_via_title_number_and_md_ref.rb

```diff
@@ -0,0 +1,24 @@
+When(/^I search the deed store$/) do
+  str_end = '/deed?md_ref=e-MD12344&title_number='
+  deed_end_point = Env.deed_api + str_end + @title_number.to_s
+  @response = HTTP.get(deed_end_point)
+
+  # Open the URL in browser to check information
+  head = 'http://10.10.10.10:9020/deed?md_ref=e-MD12344&title_number='
+  full_str = head + @deed.title_number.to_s
+  visit(full_str)
+end
+
+When(/^I search the deed store with title_number "([^"]*)"$/) do |title_number|
+  # Open the URL in browser to check information
+  head = 'http://10.10.10.10:9020/deed?md_ref=e-MD12344&title_number='
+  full_str = head + title_number.to_s
+  visit(full_str)
+end
+
+Given(/^I create the deed with title_number "([^"]*)"$/) do |title_num|
+  # Using to create deed with specific title number
+  @deed = Deed.new(1)
+  @deed.title_number = title_num
+  @deed_hash = @deed.to_hash
```

Comment:
A bit more unambiguity with the variable names ? Not sure what "full_str" and "str_end" means. Good job though :smiley: